onnx/reference/ops/op_resize.py

```diff
@@ -72,7 +72,7 @@ def _nearest_coeffs(ratio: float, mode: str = "round_prefer_floor") -> np.ndarra
     raise ValueError(f"Unexpected value {mode!r}.")
 
 
-def _cubic_coeffs(ratio: float, scale: float, A: float = -0.75) -> np.ndarray:
+def _cubic_coeffs(ratio: float, scale: float = None, A: float = -0.75) -> np.ndarray:
```

Comment:
nit: `typing.Optional[float]` would be a better annotation for `scale`. Same below.